src/graphql/types/Mutation/updateAgendaItem.ts

```
@@ -14,284 +14,281 @@
-					},
-					where: (fields, operators) => operators.eq(fields.id, currentUserId),
-				}),
-				ctx.drizzleClient.query.agendaItemsTable.findFirst({
-					columns: {
-						type: true,
-					},
+	const [currentUser, existingAgendaItem] = await Promise.all([
+		ctx.drizzleClient.query.usersTable.findFirst({
+			columns: { role: true },
+			where: (
+				fields: any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)